tests/unit/api.py

```diff
@@ -1204,6 +1204,21 @@ def test_list_projects(self, db):
             'runway',
         ]
 
+    import uuid
+
+    def test_list_networks(self, db):
+        assert json.loads(api.list_networks()) == {}
+        api.project_create('anvil-nextgen')
+        network_create_simple('netA', 'anvil-nextgen')
```

Comment:
Please add a test to output a network not assigned to any project. 
See how that shows up in the output. 
Just add another network using `network_create` instead `network_create_simple`.